qa/TL3_pipeline_zoo/test_images.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -ex
+pushd /opt/dali/docs/examples/zoo/images
+
+scripts=(
+    "decode.py --images_dir $DALI_EXTRA_PATH/db/coco/images/" \
+    "decode_and_transform_pytorch.py --landmarks_dir $DALI_EXTRA_PATH/db/face_landmark/" \
+    "decode_and_transfrom_from_json.py --coco_dir $DALI_EXTRA_PATH/db/coco/" \
+    )
+
+for SCRIPT_NAME in "${scripts[@]}"; do
+    python $SCRIPT_NAME
+    RV=$?
+    if [ $RV -gt 0 ]; then
+        echo "Failed! $SCRIPT_NAME"
+        exit $RV
+    fi
```

Comment:
I wonder if `-ex` doesn't do the job, on the other hand it could be better to be explicit.

Reply:
I mean `#/bin/bash -ex` on top of the file.